mirrors.d/almalinux.repo.slimhost.com.ua.yml

```diff
@@ -0,0 +1,12 @@
+name: almalinux.repo.slimhost.com.ua
+address:
+  http: https://almalinux.repo.slimhost.com.ua/
```

Comment:
it seems Ukraine, not Germany, hasn't got the idea of this

Reply:
It's not mine change, just from another PR. I can't to change anything where.